src/js/components/commons/nav-bar/app-menu.js

```diff
@@ -70,6 +70,14 @@ export default ({
 				<ListItem>
 					<ListItemText primary="La plateforme" />
 				</ListItem>
+				{hasOptedInForBetaTest() ? (

```

Comment:
I think we should hide (or disable ?) menu items that have not been configured.
e.g for this : if  `FORMATIONS_URL` is not set then we should hide or disable this menu item.  
This could apply to other features (e.g : services) but may be out of scope of this PR

Reply:
Done :)